### DISCUSSION TITLE
Question about Quiz 02

### DISCUSSION THREAD

**Discussion by AjeethUConn** [cooperrc/advanced-dynamics#26] [General]
I know how many degrees of freedom the system has and what my generalized coordinates are. However, I'm not sure where to start to properly solve 2 and 3. Is it as simple as taking the derivative of both of r_1 and r_2 with respect to time, or is it something like incorporating those as part of a Newton's 2nd Law equation?

**Comment by Maid-Unknown** [comment]
I was confused either, I thought Just take the derivative of that. Not sure, we can ask professor during the Thursday Question time.

**Comment by travismalave** [comment]
IMO, we would still need to determine the EoM for x since we don't have an explicit expression for it with respect to time. Then, plug in for x and determine velocity from there. That's the direction I'm going using the Lagrange method but I think similar can be done via FDB and kinetic diagrams like the prof has shown in the past. 

What throws me off is being given an equation for theta when I thought we would have to determine this as well, but maybe my choice of generalize coordinates are off...
